.github/workflows/uffizzi-preview-deploy.yml

```diff
@@ -0,0 +1,102 @@
+name: Deploy Uffizzi Preview
+
+on:
+  workflow_run:
+    workflows:
+      - "Build PR Image"
+    types:
+      - completed
```

Comment:
Since you need the previous workflow to be completed, can't we make them in the same workflow instead? The `cache-compose-file` could only start when `build-pr-image` is done.
It complexity to have two separated files